### PR TITLE
OCPBUGS#44700: ContentX for the PTP and SCTP networking docs

### PR DESCRIPTION
**Pull request by dfitzmau**
INTERNAL CHANGE THAT IS JUST EXPANDING ABBREVIATIONS. NO SME OR QE NEEDED.

Version(s):
4.12+

Issue:
[OCPBUGS-44700](https://issues.redhat.com/browse/OCPBUGS-44700)

Link to docs preview:
* [PTP guides](https://85775--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ptp/about-ptp)

### CODE REVIEW

File: modules/ptp-using-hardware-specific-nic-features.adoc
Comment (aireilly) [inline modules/ptp-using-hardware-specific-nic-features.adoc:1]: Why was this module deleted?

Comment (aireilly) [reply modules/ptp-using-hardware-specific-nic-features.adoc:1]: Ah. nevermind. You incorporated it elsewhere.

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -1497,15 +1497,15 @@ Topics:
   File: configuring-ipfailover
 - Name: Configuring system controls and interface attributes using the tuning plugin
   File: configure-syscontrols-interface-tuning-cni
-- Name: Using SCTP
+- Name: Using Stream Control Transmission Protocol (SCTP)
   File: using-sctp
   Distros: openshift-enterprise,openshift-origin
-- Name: Using PTP hardware
+- Name: Using Precision Time Protocol hardware
```
Comment (aireilly) [inline _topic_maps/_topic_map.yml:1503]: - Name: Using Precision Time Protocol (PTP) hardware

File: networking/ptp/ptp-events-rest-api-reference-v2.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="ptp-events-rest-api-reference-v2"]
-= PTP events REST API v2 reference
+= Precision Time Protocol events REST API {ptp-events-rest-api} reference
```
Comment (aireilly) [inline networking/ptp/ptp-events-rest-api-reference-v2.adoc:3]:
```suggestion
= Precision Time Protocol events REST API {ptp-events-rest-api} reference
```

Are you adding a variable for "v2"? Why?

File: networking/ptp/configuring-ptp.adoc [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
 :context: configuring-ptp
 
 toc::[]
 
 The PTP Operator adds the `NodePtpDevice.ptp.openshift.io` custom resource definition (CRD) to {product-title}.
 
-When installed, the PTP Operator searches your cluster for Precision Time Protocol (PTP) capable network devices on each node. It creates and updates a `NodePtpDevice` custom resource (CR) object for each node that provides a compatible PTP-capable network device.
+When installed, the PTP Operator searches your cluster for Precision Time Protocol (PTP) capable network devices on each node. The Operator creates and updates a `NodePtpDevice` custom resource (CR) object for each node that provides a compatible PTP-capable network device.
+
+Network interface controller (NIC) hardware with built-in PTP capabilities sometimes require a device-specific configuration. You can use hardware-specific NIC features for supported hardware with the PTP Operator by configuring a plugin in the `PtpConfig` custom resource (CR). The `linuxptp-daemon` service uses the named parameters in the `plugin` stanza to start `linuxptp` processes, `ptp4l` and `phc2sys`, based on the specific hardware configuration.
```
Comment (JoeAldinger) [inline networking/ptp/configuring-ptp.adoc:13]: Sharp and cogent, nice work.

**Review by JoeAldinger — APPROVED** [review]
/lgtm thanks Darragh good work here.

**Comment by kalexand-rh** [comment]
Don't skip 4.13.

**Comment by JoeAldinger** [comment]
I believe this needs to target 4.13 as well cc @kalexand-rh for guidance/clarity

**Pull request merged**